### DISCUSSION TITLE
Can't renew teacher benefits because ID no longer meets requirements

### DISCUSSION THREAD

**Discussion by MikeTheGreat** [community/community#45730] [GitHub Education] [answered]
Hello!

I got an email saying that I need to renew my teacher benefits for GitHub Education, which is fine.
I went to the website, snapped a picture of my school ID, and uploaded it to the GH web app.
The pre application verification step declined my ID because it couldn't find the text on the ID (it does have text, I promise :) ) and because there's no expiration date on the ID.

I've got a working college email address, I'm listed on the college website as being a teacher, and I've previously been approved for GitHub Education teacher benefits. 

Any help with getting renewed would be greatly appreciated. Thanks in advance!

**Comment by jge162** [comment, marked as answer]
Hello and good day to you @MikeTheGreat, thank you for you post. I think you will need to submit a support request as here in the discussion forum we can only talk about your issue. Go here for further assistance ->[GitHub Education support](https://support.github.com/request/education?tags=hubberfy_education) choose the option you feel best fits your situation, or choose other if it does not, Best of luck to you. 
<img width="1014" alt="Screenshot 2023-01-29 at 6 53 56 PM" src="https://user-images.githubusercontent.com/31228460/215377722-1e3af210-452c-41f5-b12f-a86f755d543e.png">

**Reply by huichen-cs** [reply]
I am in the same situation, But the support page now says that "Individual manual reviews and status updates are no longer provided", and there is no "Other" option to choose from. 

To be clear, my situation is that
1. The faculty ID has no expiration date. The best I can do is to get a validation sticker (like what students would get, but not required for faculty)
2. I have a working college email address, am listed on the college website as being a faculty member, and I've previously been approved for GitHub Education teacher benefits.

What is my option to get verified? Thanks.

**Comment by MikeTheGreat** [comment]
Thanks!

I saw the page that you listed but missed the 'Contact Support' option, and got stuck in a loop that sent me back to an earlier page.  This time I found it - thanks for the pointer!
